cs/src/Contracts/TunnelAccessControl.cs

```diff
@@ -77,16 +77,24 @@ public TunnelAccessControl(IEnumerable<TunnelAccessControlEntry> entries)
         /// <param name="scopes">List of scopes to validate.</param>
         /// <param name="validScopes">Optional subset of scopes to be considered valid;
         /// if omitted then all defined scopes are valid.</param>
+        /// <param name="allowMultiple">Whether to allow multiple space-delimited scopes in a
```

Comment:
Should this comment indicate when/why this would be beneficial?

Reply:
Added.